outlook/calendar/tool.gpt

```diff
@@ -59,7 +59,7 @@ Param: start: (Required) The start time of the event, in RFC 3339 format.
 Param: end: (Required) The end time of the event, in RFC 3339 format. When scheduling a recurring event, this should be the end time of the first event in the series.
 Param: recurrence: (Optional) If the meeting should recur, describe in plain English how often it should occur (daily, weekly, monthly, yearly) and during which date range (first and last occurrence) or how many total times the event should occur. ALWAYS include the date of the first occurrence, and optionally the date of the last occurrence.
 Param: calendar_id: The unique ID of the calendar or group to add the event to. If unset, adds the event to the default calendar.
-Param: owner_type: The type of the owner of the calendar or group. Possible values are "user" or "group". Required if calendar_id is set.
+Param: owner_type: (Required if calendar_id is set) The type of the owner of the calendar or group. Possible values are "user" or "group".
```

Comment:
nit: feels like we should be able to get by without `owner_type` if we already have a unique `calendar_id`, but I guess the SDK design doesn't make that easy. We could always inject the owner type into the ids we return from the list operation, but if the LLM can reliably figure this field out on it's own, that's probably unnecessary.

Reply:
Yeah after I rearranged the prompt, the LLM seems to not have an issue figuring this out. Imo it's best to leave it a it is.